File: sprint4/By lessons/lesson2code1/lesson2code1.py
# Функция bucket вычисляет номер корзины для указанного фрукта
def bucket(fruit):
    if fruit == "яблоко":
        return 0
    elif fruit == "слива":
        return 1
    elif fruit == "груша":
        return 2
    else:
        raise ValueError("Неизвестный фрукт")

File: sprint4/By lessons/lesson2code1/lesson2code1.go
// Функция bucket вычисляет номер корзины для указанного фрукта
func bucket(fruit string) int {
    switch fruit {
    case "яблоко":
        return 0
    case "слива":
        return 1
    case "груша":
        return 2
    default:
        panic("Неизвестный фрукт")
    }
}
